packages/parser/lib/css_types.re

```diff
@@ -2,6 +2,8 @@
 
 module Location = Ppxlib.Location;
 
+let nesting_level = ref(0);
```

Comment:
Maybe call it selector_nesting_level?

Probably nice to add a comment explaining what it is also.